python/ct/client/async_log_client_test.py

```diff
@@ -113,7 +114,7 @@ def request(self, method, uri):
 
     class FakeDB(object):
         def scan_entries(self, first, last):
-            raise database.KeyError
+            raise database.KeyError("boom!")
```

Comment:
I changed it, so we raise actual instance of Exception. Also it's nice to have boom! so we know that it's test exception (the same pattern is in monitor_test)